src/settings/BatchGroupConfigurationSettings/ExportVoucherButton/ExportVoucherButton.js

```diff
@@ -0,0 +1,67 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import saveAs from 'file-saver';
+
+import {
+  stripesConnect,
+  stripesShape,
+} from '@folio/stripes/core';
+import { IconButton } from '@folio/stripes/components';
+import { useShowCallout } from '@folio/stripes-acq-components';
+
+import { BATCH_VOUCHERS_API } from '../../../common/constants';
+
+const ExportVoucherButton = ({ batchVoucherId, format, stripes }) => {
+  const showCallout = useShowCallout();
+  const downloadBatchVouchers = useCallback(
+    async () => {
+      try {
+        const httpHeaders = {
+          'X-Okapi-Tenant': stripes.okapi.tenant,
```

Comment:
as I remember there is a hook for okapi

Reply:
found only one in [data-import](https://github.com/folio-org/ui-data-import/blob/61db482c74a3b4cb1a6274138a537878fa64ea35/src/utils/createOkapiHeaders.js#L7)